pyproject.toml

```diff
@@ -45,6 +45,8 @@ classifiers = [
     'Environment :: Console',
     'Environment :: MacOS X',
     'Framework :: Hypothesis',
+    'Framework :: Pydantic',
+    'Framework :: Pydantic :: 1',
```

Comment:
I am not sure that we should include a classifier with the version in Pydantic.

I can't see it in [Django](https://github.com/django/django/blob/main/setup.cfg#L13), [FastAPI](https://github.com/tiangolo/fastapi/blob/master/pyproject.toml#L29), and [Flask](https://github.com/pallets/flask/blob/main/pyproject.toml#L11).

Reply:
I see we can delete it 🐛 

```suggestion
```

Reply:
good catch, agreed.